platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/offline/OfflineRegionError.java

```diff
@@ -0,0 +1,47 @@
+package com.mapbox.mapboxsdk.offline;
+
+/**
+ * Created by antonio on 2/21/16.
+ */
+public class OfflineRegionError {
+
+    /**
+     * Success = 1
```

Comment:
My concern was more about tying these to specific integers that must match `Response::Error::Reason`. We are fairly likely to change `Response::Error::Reason` (to remove the current gap in numbering, for instance), which would break Java code that compared with, say, `REASON_OTHER`.